showyourwork/workflow/scripts/arxiv.py

```diff
@@ -26,8 +30,22 @@
         f"{ms_name}.fdb_latexmk",
     ]
 
-    # Tar up everything in the src/tex directory
-    with tarfile.open("arxiv.tar.gz", "w:gz") as tarball:
-        for file in paths.user().compile.rglob("*"):
-            if file.name not in exclude and file.suffix != ".bib":
-                tarball.add(file, arcname=file.name)
+    with TemporaryDirectory() as tmpdir:
+        # First, copy to a temporary directory, excluding files
+        shutil.copytree(paths.user().compile, tmpdir, dirs_exist_ok=True)
```

Comment:
This copy isn't necessary unless the preprocessing is enabled, I'd move the check above so that we only do this extra copy if needed.

Reply:
Sounds good. Will add.